app/templates/components/nav-bar.hbs

```diff
@@ -68,7 +68,7 @@
         </a>
       {{/if}}
       <div class="item">
-        <a class="ui blue button" style="text-transform: uppercase; font-weight: 400;" href="{{href-to 'create'}}">
+        <a class="mini ui blue button button-text {{if device.isMobile 'small text'}}" style="text-transform: uppercase; font-weight: 400;" href="{{href-to 'create'}}">
```

Comment:
Don't make the button mini, remove the mini class. I tested and it was fine. Small text class is enough for making it responsive across all portrait modes.